packages/extension-browser/package.json

```diff
@@ -6,7 +6,7 @@
     ],
     "timeout": "1m"
   },
-  "bundleSize": 615000,
+  "bundleSize": 665000,
```

Comment:
50K more? This doesn't make a lot of sense for the code that's being added. Maybe a wrong import that's pulling more stuff than needed?

Reply:
yep, it is weird, I'm taking a look to the changes.

Reply:
I revert this change. The change size was something unrelated with my PR